src/App.js

```diff
@@ -1,25 +1,9 @@
-import logo from './logo.svg';
-import './App.css';
+import Shared from "./pages/shared.js";
+import "./style/global.css";
+import "./style/reset.css";
```

Comment:
```suggestion
import "./style/reset.css";
import "./style/global.css";
```

css 파일도 import 한 순서에 따라 우선순위가 적용 되기 때문에
global에서 설정한 스타일이 reset 에서 다시 설정 될 수 있어서
보통 일반적으로는 reset을 먼저 import 합니다!